test/test_jit.py

```diff
@@ -1348,7 +1347,7 @@ def func(x) -> (head):
         '''
         x = Variable(torch.rand(10).float(), requires_grad=True)
         outputs = x[:5]
-        self.checkScript(script, [x], [outputs], False)
+        self.checkScript(script, [x], outputs, False)
```

Comment:
Why was this changed?

Reply:
I think it's because GraphExecutors used to return tuples, but now return a single Variable, or a tuple of Variables that has at lest 2 elements.